diagnostics/model_test/mnist_cnn.json

```diff
@@ -0,0 +1,10 @@
+{
+    "model_name": "mnist_cnn",
+    "frozen_graph_location": "/nfs/site/home/mingshan/frozen_graphs/frozen_mnist.pb",
+    "input_tensor_name": "import/Placeholder:0",
+    "output_tensor_name":"import/fc2/yconv:0",
+    "l1_norm_threshold": 0.01,
+    "l2_norm_threshold": 0.01,
+    "inf_norm_threshold": 0.01,
+    "input_dimension" : [784]
+}
```

Comment:
Networks could have >1 inputs. So input dims might need to be specified as list of list. same with input_tensor_name. In this case, the input dim should be [[1,784]]. note if user should specify batch size here, instead of us adding [1] as we currently have in this line:
```random_input = np.random.random_sample(tuple([1] + input_dimension))```
Also we can assert length of this list equals length of list of input_tensor_names.

there might also be multiple outputs, but for now we can assume 1 out.

Reply:
Hi @sayantan-nervana , I just updated the script to support multiple inputs, and also added one batch_size field, so for input_dimension, user should specify [[784]] in this case, and batch_size =1. 

Also, FYI, I didn't format the code this time as well, maybe you can use this to check whether the formatter is working or not.